core/src/main/java/org/kohsuke/stapler/RequestImpl.java

```diff
@@ -734,14 +734,23 @@ public Object convertJSON(Object o) {
 
             if (l==null) {// single value conversion
                 Converter converter = Stapler.lookupConverter(type);
-                if (converter==null)
+                if (converter==null) {
```

Comment:
```suggestion
                if (converter == null) {
```

Reply:
I will change, I simply keep the actual formatting

Reply:
Just old nonstandard code style from @kohsuke; I generally clean up when modifying lines anyway (no extra diff penalty). Does not matter.